src/sentry/static/sentry/app/views/settings/types.ts

```diff
@@ -41,6 +41,11 @@ export type NavigationItem = {
    * Returns the text of the badge to render next to the navigation.
    */
   badge?: (opts: NavigationGroupProps) => string | number | null;
+
```

Comment:
tiny tiny nit, can you remove this space. I don't think any of the comments before have it